File: vb.net/med_com_rubik.vb
' https://www.codingame.com/training/medium/rubik%C2%AE

Module Solution
    Sub Main ()
        Dim N as Integer
        N = Console.ReadLine()
        Dim ans as Integer
        If (N > 1) Then
            ans = 6 * N * (N - 2) + 8
        Else
            ans = 1
        End If
        Console.WriteLine(ans)
    End Sub
End Module
' Write an action using Console.WriteLine()
' To debug: Console.Error.WriteLine("Debug messages...")

File: c++/med_com_rubik.cpp
// https://www.codingame.com/training/medium/rubik%C2%AE

#include <iostream>
using namespace std;

int main()
{
    int n;
    std::cin>>n;
    std::cout<<(n>1?6*n*(n-2)+8:1)<<"\n";
}
// Write an action using cout. DON'T FORGET THE "<< endl"
// To debug: cerr << "Debug messages..." << endl;

// #include <stdlib.h>
// main(){system("python3 -c\"n=int(input());print(6*n*(n-2)+8 if n>1 else 1)\"");}
